<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Zapisz</class>
 <widget class="QWidget" name="Zapisz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zapisywanie</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Nazwa</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Has≈Ço</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nameLine"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="loginLine"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="passwordLine"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="zapiszButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Zapisz</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="zamknijButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>Zamknij</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
